website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -75,24 +76,24 @@ The initialization of the sweep and the first 5 evaluations (out of 100) look li
 [HYDRA] Sweep output dir: multirun/2020-03-04/17-53-29
 [HYDRA] Launching 10 jobs locally
 [HYDRA] 	#0 : db=mnist batch_size=8 lr=0.032 dropout=0.5
-[__main__][INFO] - dummy_training(dropout=0.500, lr=0.032, db=mnist, batch_size=8) = 5.258
+[__main__][INFO] - my_app.py(dropout=0.500, lr=0.032, db=mnist, batch_size=8) = 5.258
 [HYDRA] 	#1 : db=mnist batch_size=16 lr=0.035 dropout=0.714
-[__main__][INFO] - dummy_training(dropout=0.714, lr=0.035, db=mnist, batch_size=16) = 13.469
+[__main__][INFO] - my_app.py(dropout=0.714, lr=0.035, db=mnist, batch_size=16) = 13.469
 [HYDRA] 	#2 : db=cifar batch_size=8 lr=0.053 dropout=0.408
-[__main__][INFO] - dummy_training(dropout=0.408, lr=0.053, db=cifar, batch_size=8) = 4.145
+[__main__][INFO] - my_app.py(dropout=0.408, lr=0.053, db=cifar, batch_size=8) = 4.145
 [HYDRA] 	#3 : db=cifar batch_size=8 lr=0.012 dropout=0.305
-[__main__][INFO] - dummy_training(dropout=0.305, lr=0.012, db=cifar, batch_size=8) = 4.133
+[__main__][INFO] - my_app.py(dropout=0.305, lr=0.012, db=cifar, batch_size=8) = 4.133
 [HYDRA] 	#4 : db=mnist batch_size=4 lr=0.030 dropout=0.204
-[__main__][INFO] - dummy_training(dropout=0.204, lr=0.030, db=mnist, batch_size=4) = 1.216
+[__main__][INFO] - my_app.py(dropout=0.204, lr=0.030, db=mnist, batch_size=4) = 1.216
 ```
 
 
 and the final 2 evaluations look like this:
 ```text
 [HYDRA] 	#8 : db=mnist batch_size=4 lr=0.094 dropout=0.381
-[__main__][INFO] - dummy_training(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
+[__main__][INFO] - my_app.py(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
 [HYDRA] 	#9 : db=mnist batch_size=4 lr=0.094 dropout=0.381
-[__main__][INFO] - dummy_training(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
+[__main__][INFO] - my_app.py(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
```

Comment:
the output really contains `my_app.py` ?

Reply:
😂 that is a copy paste oversight. will fix